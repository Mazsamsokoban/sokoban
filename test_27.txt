loadtest 27
move w1 right
showmaze