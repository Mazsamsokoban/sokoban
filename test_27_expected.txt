f0:w1 f1:box f2:w2 f3:w3 f4:null 

f0:w1 f1:box f2:w2 f3:w3 f4:null 

